javascript/javascript1/week2/script_1.js

```diff
@@ -0,0 +1,78 @@
+//! FreeCodeCamp Link: https://www.freecodecamp.org/catalina-tech
+
+//* Flight booking fullname function
+
+//*To add the Lady or Lord in front of the fullName, I created two radio buttons, one for female and another one for male
+//* If you select useFormalName && male, it will appear Lord at the beggining
+//* If you select userFormalName && Female, it will appear Lady at the beginning
+//* This is how I fixed that the getFullname also works for women
+//!Codepen link: https://codepen.io/Catalina-Bernal/pen/MWRdorw?editors=1010
+
+
+function getFullname(firstName, surName, useFormalName, male, female) {
+
+  let fullName = firstName + " " + surName;
```

Comment:
For these kinds of string concatenations I would suggest using template literals `${firstName} ${surName}`